reference/CloudPay-API-Specification.yaml

```diff
@@ -2492,9 +2483,7 @@ paths:
                     Style: null
                     Amount: 9.99
                     DisplayOrder: 0
-                        Entitlements: [
-                          "paid"
-                        ]
+                    Entitlements: ["paid"]
```

Comment:
you have to add this field to the schema too otherwise this property will not be present in here:

<img width="615" alt="Screenshot 2024-03-21 at 12 23 35" src="https://github.com/StreamAMG/cloudpay-documentation-portal/assets/149777948/1463dc42-e199-4ff7-814a-5f6c018b11ea">

Reply:
Done